pyproject.toml

```
@@ -9,7 +9,7 @@ packages = [{include = "ot_orchestration", from = "src"}]
 
 [tool.poetry.dependencies]
 python = "^3.10, <3.11"
-apache-airflow = {extras = ["apache-beam", "async", "celery", "google", "kubernetes", "openlineage", "postgress", "sftp"], version = "^2.9.2"}
```

How strange , that poetry did allow for this

---

Maybe it is failing silently on that extra install?